src/modules/75-cf/components/FlagArchiving/CannotArchiveWarning.tsx

```diff
@@ -5,19 +5,20 @@ import { Link, useParams } from 'react-router-dom'
 import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
 import routes from '@common/RouteDefinitions'
 import { useStrings } from 'framework/strings'
-import type { Feature, Prerequisite } from 'services/cf'
+import type { Prerequisite } from 'services/cf'
 import { ItemContainer } from '../ItemContainer/ItemContainer'
 
 export interface CannotArchiveWarningProps {
-  flagData: Feature
+  flagName: string
+  prerequisites: Prerequisite[]
 }
 
-const CannotArchiveWarning: FC<CannotArchiveWarningProps> = ({ flagData }) => {
+const CannotArchiveWarning: FC<CannotArchiveWarningProps> = ({ flagName, prerequisites }) => {
   const { getString } = useStrings()
   const { withActiveEnvironment } = useActiveEnvironment()
   const { projectIdentifier, orgIdentifier, accountId } = useParams<Record<string, string>>()
 
-  const prerequisiteFlagIds = flagData?.prerequisites?.map((flag: Prerequisite) => {
+  const prerequisiteFlagIds = prerequisites?.map(flag => {
```

Comment:
You know this will exist and be an array, so you don't need this `?`. It's probably the reason Istanbul things there's a branch
```suggestion
  const prerequisiteFlagIds = prerequisites.map(flag => {
```